src/firefly/html/demo/ffapi-coverage-test.html

```diff
@@ -71,13 +71,23 @@
 
             firefly.showCoverage('coverageHere',
                     {
-                        gRIdOn:true,
+                        gridOn:true,
                         symbol: {aTable: 'X', bTable: 'arrow'},
                         symbolSize: {aTable: 3, bTable: 12},
                         color: {aTable: 'blue', bTable: 'yellow'},
-//                        overlayPosition : '149.9;68.8;EQ_J2000',
-                        OVErlayPosition : '149.08;68.739;EQ_J2000',
-                        useHiPS: true
+                        overlayPosition : '149.08;68.739;EQ_J2000',
+
+                        useHiPS: true,
+                        useAllSkyFitsForPlus180 : true,
+                        // fovDegFallOver: .02,
+                        fovDegFallOver: .5, // 720 arcsec
+                        hipsSourceURL : 'http://alasky.u-strasbg.fr/DSS/DSSColor', // url
+                        // hipsSourceURL : 'http://alasky.u-strasbg.fr/Planets/CDS_P_Earth_BlueMarble/', // url
+                        imageSourceParams: {
+                            Service : 'WISE',
```

Comment:
is this json attribute (`imageSourceParams`) defining the fits image dataset when zooming in/out on coverage?

Reply:
> and i'm not sure if it was working previously but footprint overlay doesn't work on hips maps, so either we need to raise a ticket ot fix it or disable the feature when hips map is active.

That is being worked on in
https://github.com/Caltech-IPAC/firefly/pull/558

Reply:
Great!

Reply:
Image is for zooming in and hips is for zooming out.  When come first is based on how big the search area is and `fovDegFallOver`

Reply:
I guess that’s where we will need in the future some logic to decide on which image to show related to hips map - see IRSA-1586. Currently is fixed to WISE, correct?

Reply:
From Gator's perspective, I think it should provide the parameters.  When loading catalogs directly in Firefly, I guess we could have some logic to guess.  However, that is not what we have done in the pass.  It would be very easy to do some basic guessing. 
>  wise on wise, 2 mass on 2 mass, x on x, anything else on wise.
But as I said firefly is not done that before.